bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/TTZJets/TTZJets_5f_LO_MLM/ttZ01j_5f_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+generate p p > t t~ z, z > j j / h @0
+add process p p > t t~ z, z > vl vl~ / h @1
+add process p p > t t~ ell+ ell- / h @2
+add process p p > t t~ z j, z > j j /h @3
+add process p p > t t~ z j, z > vl vl~ / h @4
+add process p p > t t~ ell+ ell- j / h @5
+
+output ttZ01j_5f -nojpeg
```

Comment:
I would suggest to separate them as what we do for other processes for Run3

Reply:
This is a bit ill defined. you will have double countings from ttZ->ll